app/models/account.rb

```diff
@@ -60,6 +60,7 @@ class Account < ApplicationRecord
   scope :buyers, -> { where(provider: false, buyer: true) }
 
   scope :without_deleted, ->(without = true) { where(deleted_at: nil) if without }
+  scope :not_master, -> { where(master: nil) }
```

Comment:
It can be  0 too for false.

Reply:
not the same, `0` looks like this:
`SELECT `accounts`.* FROM `accounts` WHERE `accounts`.`master` = 0"`

and `nil` like this: 

`"SELECT `accounts`.* FROM `accounts` WHERE `accounts`.`master` IS NULL"`

Reply:
Sure but it can be `0` or `nil`

Reply:
I think you can do it `master != 1` on babysqueel

Reply:
`SELECT COUNT(*) FROM `accounts` WHERE (`accounts`.`master` != 1)`

does not work properly, it returns 0.

Reply:
@hallelujah He can't 😞 
![image](https://user-images.githubusercontent.com/11318903/47501083-df54b580-d864-11e8-9d86-d8d948accdcb.png)